src/components/dropdown-menu/ResponsiveMenuContainer.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+import * as React from 'react';
+
+import IconClose from '../../icons/general/IconClose';
+import PrimaryButton from '../../components/primary-button';
+
+import { bdlGray62 } from '../../styles/variables';
+import Button from '../../components/button/Button';
+import './ResponsiveMenuContainer.scss';
+
+export type ResponsiveMenuProps = {
+    buttonText?: string,
+    children?: React.Node,
+    className?: string,
+    onButtonClick?: Function,
+    onCloseClick?: Function,
+    reverseTitle: boolean,
+    subtitle?: string,
+    title?: string,
+};
+
+const ResponsiveMenuContainer = (props: ResponsiveMenuProps) => {
+    const { children, title, subtitle, buttonText, reverseTitle, onButtonClick, onCloseClick, ...rest } = props;
+    const primaryTitle = title && <div className="menu-header-title">{title}</div>;
+    const secondaryTitle = subtitle && <div className="menu-header-subtitle">{subtitle}</div>;
+
+    const titleComponents = [secondaryTitle, primaryTitle];
+
+    const btnProps = {
+        onClick: onButtonClick,
+    };
+
+    const closeBtnProps = {
+        onClick: onCloseClick,
+    };
+
+    return (
+        <div className="menu-container" {...rest}>
```

Comment:
Do these class names conflict with any other components? Can we use SUIT naming conventions?